src/sentry/receivers/onboarding.py

```diff
@@ -388,6 +381,45 @@ def record_user_context_received(project, event, **kwargs):
 
 
 event_processed.connect(record_user_context_received, weak=False)
+
+
+def record_event_with_first_minified_stack_trace_for_project(project, event, **kwargs):
+    # We only want to record events from projects created after 2022-12-09,
+    # otherwise amplitude would receive a large amount of data in a short period of time
+    if project.date_added < datetime.datetime(2022, 12, 9, tzinfo=pytz.utc):
+        return
+
+    if not has_event_minified_stack_trace(event):
+        return
+
+    try:
+        user: APIUser = Organization.objects.get(id=project.organization_id).get_default_owner()
+    except IndexError:
+        logging.getLogger("sentry").warning(
+            "Cannot record first event for organization (%s) due to missing owners",
+            project.organization_id,
+        )
+        return
+
+    url = None
+
+    # Check for the event url
+    for key, value in event.tags:
+        if key == "url":
+            url = value
+            break
+
+    analytics.record(
+        "first_event_with_minified_stack_trace_for_project.sent",
+        user_id=user.id,
+        organization_id=project.organization_id,
+        project_id=project.id,
+        platform=event.platform,
+        url=url,
```

Comment:
```suggestion
        url=dict(event.tags).get("url", None),
```

Reply:
sweet!

Reply:
This works under the assumption that event.tags is not None, do you know if it is always not None?

Reply:
I don't know about that and would have to check.

Reply:
@andriisoldatenko pointed me to this file https://github.com/getsentry/sentry/blob/828c5fcdd8d7c1f9d1241d5f06c768ffd5cbcafc/src/sentry/eventstore/models.py#L119-L145.

So it can be an empty list but never `None`.

in this case, the implementation shall work fine